src/brod_producer.erl

```diff
@@ -36,7 +36,7 @@
         , handle_info/2
         , init/1
         , terminate/2
-        , format_status/2
```

Comment:
please keep `format_status/2` for older version OTP

Reply:
To do this we need to remove `warnings_as_errors` from the compilation options, as Erlang/OTP complains then the process stops.

Reply:
I can add an OTP_RELEASE exception, if that's what you mean.

Reply:
b38712d